### ISSUE TITLE
message from native to Flutter on a non-platform thread

### ISSUE THREAD

**Issue by Maatteogekko** [mrcendre/motion#18]
In debug mode, running on a real iPhone 12 mini, I get this message in console:

```
The 'me.cendre.motion/gyroscope' channel sent a message from native to Flutter on a non-platform thread. Platform channel messages must be sent on the platform thread. Failure to do so may result in data loss or crashes, and must be fixed in the plugin or application code creating that channel.
See https://docs.flutter.dev/platform-integration/platform-channels#channels-and-platform-threading for more information
```

I can't reproduce it consistently and there is no effect visible in the app. Still, this may need to be addressed

**Comment by mrcendre** [comment]
Hi @Maatteogekko, thanks for the report ! 

This was indeed a known error, which is now fixed on the `master` branch.

It is now solved in the latest plugin version ([1.4.1](https://pub.dev/packages/motion/changelog#141)). Please upgrade and let me know if it solves your issue. If not, I will reopen the issue. Thank you !

**Comment by mrcendre** [comment]
Hi @Maatteogekko, thanks for the report ! 

This was indeed a known error, which is now solved in the latest plugin version ([1.4.1](https://pub.dev/packages/motion/changelog#141)). 

Please upgrade and let me know if it solves your issue. If not, I will reopen the issue. Thank you !
